utils/ccomp.ml

```diff
@@ -38,20 +38,21 @@ let run_command cmdline = ignore(command cmdline)
    between 70000 and 80000 for macOS).
 *)
 
-let build_diversion lst =
+let build_response_file lst =
   let (responsefile, oc) = Filename.open_temp_file "camlresp" "" in
   List.iter (fun f -> Printf.fprintf oc "%s\n" f) lst;
   close_out oc;
   at_exit (fun () -> Misc.remove_file responsefile);
   "@" ^ responsefile
 
-let quote_files lst =
+let quote_files ?(response_files = true) lst =
```

Comment:
Why is this an optional argument? It looks like a correctness hazard from a distance.

Reply:
Just lazyness. I pushed a commit making this argument mandatory and adapting every callsite.